bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v1/TitleImplTest.java

```diff
@@ -1,5 +1,5 @@
 /*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
- ~ Copyright 2017 Adobe Systems Incorporated
+ ~ Copyright 2019 Adobe Systems Incorporated
```

Comment:
@bpauli why bump the copyright year for some classes and not others, as I understand the initial year should remain? I also notice you update the headers for some and not others to remove "Adobe Systems Incorporated", would probably be cleaner to update them all at once in a dedicated task.